src/disclosure-manager/DisclosureManagerHeaderAdapter.jsx

```diff
@@ -0,0 +1,31 @@
+import { useLayoutEffect, useContext } from 'react';
+import PropTypes from 'prop-types';
+import DisclosureManagerHeaderAdapterContext from './DisclosureManagerHeaderAdapterContext';
```

Comment:
This is one-to-one beside the `DisclosureManagerHeaderAdapterContext` import. Can we reuse the existing implementation?

Reply:
Since we will be deprecating the existing packages, this is getting moved over wholesale.